library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx

```diff
@@ -442,20 +442,21 @@ class vtkF3DAssimpImporter::vtkInternals
 
       if (face.mNumIndices == 1)
       {
-        vtkIdType vId = face.mIndices[0];
+        vtkIdType vId = static_cast<vtkIdType>(face.mIndices[0]);
```

Comment:
Would there be a way to warn for cast like this, even on x64 ?

Reply:
No, we need to have a 32 bits CI, but I don't think it's high priority

Reply:
yup